docs/public/content/vuln_db/manuals/kb_importer.md

```diff
@@ -0,0 +1,120 @@
+# KB(Knowledge Base) Importer
+
+The vulnerability database contains known vulnerabilities for which a fix exists. In fact, the fix (commit fix in the vulnerable library project) of the vulnerability must be provided in order to add a vulnerability to the database.
+
+Each vulnerability in the database is characterized by the set of program constructs that were changed in order to fix it.
+
+!!! info "Program Construct"
+        A program construct (or simply construct) is a structural element of the source code characterized by a type (e.g., `package`, `class`, `constructor`, `method`), a language (e.g., Java, Python), and a unique identifier (e.g., the fully-qualified name).
+
+!!! info "Construct Change"
+        A construct change is characterized by a change operation (i.e., addition, deletion or modification) on a construct. In case of modifications the vulnerability database also contains the abstract syntax tree (AST) of the construct before and after the fix, i.e., the AST of the vulnerable and fixed construct.
+
+The import of such contruct changes into vulnerability database can be done using the [kb-importer](https://github.com/eclipse/steady/tree/master/kb-importer) module.
```

Comment:
[kb-importer](https://github.com/eclipse/steady/tree/master/kb-importer) is used to import such construct changes into the Steady backend.